src/agent/configuration_parser/include/configuration_parser.hpp

```diff
@@ -82,12 +91,22 @@ namespace configuration
         std::optional<T> GetConfig(Keys... keys) const
         {
             YAML::Node current = YAML::Clone(m_config);
+            bool should_parse_size = false;
 
             try
             {
                 (
-                    [&current](const auto& key)
+                    [&current, &should_parse_size](const auto& key)
                     {
+                        if constexpr (std::is_convertible_v<decltype(key), std::string_view>)
+                        {
+                            // This is a workaround for parsing size units
```

Comment:
Please, open an issue to improve this.

Reply:
@TomasTurina  Issue created : https://github.com/wazuh/wazuh-agent/issues/447